root/usr/lib64/ruby/vendor_ruby/discovery.rb

```diff
@@ -158,7 +158,7 @@ def write_tui result = 'success', code, body
 
 def upload(uri = discover_server, type = proxy_type, custom_facts = {})
   unless uri
-    log_err "Could not determine Foreman instance, add foreman.url or proxy.url kernel command parameter"
+    log_err "Could not determine instance type, add foreman.url or proxy.url kernel command parameter"
```

Comment:
I don't follow. Branding is one thing, but I'd rather stick with configuration options. We are also not renaming all our "foreman_xyz" settings in core. I'd keep it.